en/core-api/e-wallet.md

```diff
@@ -1,76 +1,88 @@
-#### **API-Request**
+To integrate with *E-Wallet* Payment method, follow the steps given below.
+
+### 1. Sending Transaction Data to API Charge
+API request should be done from merchant backend to acquire QR code and deeplink to Gojek app. The table given below describes the various elements required for sending the transaction data to the *Charge API*.  
 
-*This is an example in Curl, please implement according to your backend language, you can switch to other language on the "tab" above. (you can also check our [available language libraries](/en/technical-reference/library-plugin.md))*
+| Requirement    | Description                                                  |
+| -------------- | ------------------------------------------------------------ |
+| Server Key     | The server key. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys). |
+| `order_id`     | The order_id of the transaction.                             |
+| `gross_amount` | The total amount of transaction.                             |
+| `payment_type` | The payment method. Note: Here, set to `gopay`               |
 
 #### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://api.sandbox.midtrans.com/v2/charge`
-API endpoint (Production) | `https://api.midtrans.com/v2/charge`
 
-#### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
+| Environment | Method | URL                                        |
+| ----------- | ------ | ------------------------------------------ |
+| Sandbox     | POST   | https://api.sandbox.midtrans.com/v2/charge |
+| Production  | POST   | https://api.midtrans.com/v2/charge         |
+
+#### Headers
+
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
+
+**AUTH_STRING**: Base64(`ServerKey + :`)<br>Midtrans API validates HTTP request by using Basic Authentication method. The username is your *Server Key* while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by a colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
+
+?> ***Note***: *Server Key* is required to authenticate the request. For more details, refer to [HTTPS Header](https://api-docs.midtrans.com/#http-s-header).<br>
```

Comment:
`Request Details` and `Headers` section is only relevant if the request being made via CURL. For other language tabs which you have added, it is not relevant to show `Request Details` and `Headers`. So please move it into the CURL tabs.